### ISSUE TITLE
Wrap NFT Price

### ISSUE THREAD

**Issue by dollarmonopoly** [verynifty/veryniftyproposals#2]
The amount needed to wrap a foreign NFT, into a vNFT on VeryNifty, should be based on the total supply of the $MUSE token: 0xb6ca7399b4f9ca56fc27cbff44f4d2e4eef1fc81 (“Muse”) and remain at a constant percentage (0.33%), as opposed to a set number of Muse. 

For instance, the current total supply of Muse is 60,000 (as of Oct. 27, 2020) and the amount to wrap a foreign NFT is 200 Muse, therefore the amount to wrap a foreign NFT, into a vNFT to be used on VeryNifty, is 0.33% of the current total supply of Muse. 

The percentage of 0.33% of the total supply being required to wrap a NFT will stay the same, with the corresponding Muse amount fluctuating accordingly.

Example: When the total supply of Muse rises to the hard cap of 1,000,000 tokens, in order to wrap a NFT, it would require 0.33% of the current total supply, or 3,300 Muse, to be used.

**Comment by been9** [comment]
It's a good idea but it will make new users playing the game very difficult because when muse say reaches half of token supply say 500k, amount of muse needed to just enter the game will be around 1650 muse which will be very high. So it should be 0.33% supply 100 muse needed to wrap, 0.66% 200, 1.22% supply 300, 2.44 percent supply 400, 4.88 percent supply 500, 9.76 percent supply 600, 19.52 % supply 700, 39.04 percent supply 800, 78.08 percent supply 900 and than 1000 muse at 90% plus supply

**Comment by dollarmonopoly** [comment]
I think the price to wrap an nft should be (increasingly) high but 0.33% of total supply is too high, you're right. A player can enter the game by participating in the giveaways (airdrop, etc.), buying on OpenSea, or minting through pooling their $muse. I'm sure there are other ways coming as well. 

Can you explain what you mean by "0.33% supply 100 muse, 0.66% 200..."? Do you mean that as the total supply increases to 100%, the muse amount needed to wrap an nft gets higher? I think there should be a standard on how this number is risen, as with all the tokenomics. If the number of required anything is changed too often, it can cause distrust.

**Comment by been9** [comment]
Yes that's what I mean it should be geometric progression 2x as I mentioned with supply of muse in circulation

**Comment by surfer77** [comment]
This requires a lot of manual intervention.

The most exciting feature for me and Jules when we started this was the "wrapping" capabilities but so far people only used it as a hack to mint more muse at a better cost. (no one wrapped since we raised the price)

We are rethinking the use of this feature.
